modules/bogo/includes/Ajax.php

```
@@ -56,6 +56,44 @@ public function register_hooks(): void {
+		if ( class_exists( 'STOREGROWTH\SPSB\Modules\BoGo\BogoDataManager' ) ) {
+			// Return a wrapper that uses the new data manager
+			return new class {
+				public function create( $data ) {
+					return \STOREGROWTH\SPSB\Modules\BoGo\BogoDataManager::create_global_offer( $data );
+				}
+				
+				public function get_item( $id ) {
+					return \STOREGROWTH\SPSB\Modules\BoGo\BogoDataManager::get_bogo_offer( $id );
+				}
+				
+				public function get_items( $args = array() ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$args'. (Unused Code Rules)

(UnusedFormalParameter)